tests/test_pump.py

```diff
@@ -13,7 +12,8 @@ class PumpTest(unittest.TestCase):
     def setUp(self):
         self.mock_pi_io = mock.Mock()
         self.mock_clock = mock.Mock(spec=clock.Clock)
-        self.pump = pump.Pump(self.mock_pi_io, self.mock_clock)
+        self.pump_pin = 0
```

Comment:
Suggest using a different value to make it clearer that this is a GPIO pin. On a Pi3, these can be 2 to 27. Plus I'm generally nervous about using `0` because I feel like there's potential for the test to pass by mistake, since it's easier for something to accidentally get set to `0` than say `6`.